setup.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
+echo "install poetry if not in path"
+if ! command -v poetry &> /dev/null
+then
+    echo "poetry is missing. Installing ..."
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
```

Comment:
executing some remote file is not exactly the safest way of installing things. Any other options of getting poetry, pip install poetry maybe? :-)